libs/brackets-server/embedded-ext/hirse.beautify/main.js

```diff
@@ -0,0 +1,111 @@
+define(function (require) {
+    'use strict';
+
+    var PREFIX = 'hirse.beautify';
+    var COMMAND_ID = PREFIX + '.beautify';
+
+    /* beautify preserve:start */
+    var CommandManager     = brackets.getModule('command/CommandManager');
+    var Menus              = brackets.getModule('command/Menus');
+    var DocumentManager    = brackets.getModule('document/DocumentManager');
+    var Editor             = brackets.getModule('editor/Editor').Editor;
+    var EditorManager      = brackets.getModule('editor/EditorManager');
+    var DefaultDialogs     = brackets.getModule('widgets/DefaultDialogs');
+    var Dialogs            = brackets.getModule('widgets/Dialogs');
+    /* beautify preserve:end */
+
+    var Strings = require('strings');
+    var beautifiers = {
+        js: require('thirdparty/beautify').js_beautify,
+        css: require('thirdparty/beautify-css').css_beautify,
+        html: require('thirdparty/beautify-html').html_beautify
+    };
+
+    var settings = JSON.parse(require('text!default.jsbeautifyrc'));
+    var keyBindings = [
+        {
+            key: 'Ctrl-Shift-L',
+            platform: 'win'
+        }, {
+            key: 'Ctrl-Alt-B',
+            platform: 'win'
+        }, {
+            key: 'Cmd-Shift-L',
+            platform: 'mac'
+        }, {
+            key: 'Ctrl-Alt-B'
+        }
+    ];
+
+    function batchUpdate(formattedText, isSelection) {
+        var editor = EditorManager.getCurrentFullEditor();
+        var cursorPos = editor.getCursorPos();
+        var scrollPos = editor.getScrollPos();
+        var doc = DocumentManager.getCurrentDocument();
+        var selection = editor.getSelection();
+        doc.batchOperation(function () {
+            if (isSelection) {
+                doc.replaceRange(formattedText, selection.start, selection.end);
+            } else {
+                doc.setText(formattedText);
+            }
+            editor.setCursorPos(cursorPos);
+            editor.setScrollPos(scrollPos.x, scrollPos.y);
+        });
+    }
+
+    function format() {
+        var beautifierType;
+        var document = DocumentManager.getCurrentDocument();
+        switch (document.getLanguage().getId()) {
+        case 'javascript':
+        case 'json':
+            beautifierType = 'js';
+            break;
+        case 'html':
+        case 'xml':
+        case 'svg':
+        case 'php':
+        case 'ejs':
+        case 'handlebars':
+            beautifierType = 'html';
+            break;
+        case 'css':
+        case 'scss':
+        case 'less':
+            beautifierType = 'css';
+            break;
+        default:
+            Dialogs.showModalDialog(DefaultDialogs.DIALOG_ID_ERROR, Strings.UNSUPPORTED_TITLE, Strings.UNSUPPORTED_MESSAGE);
+            return;
+        }
+
+        var unformattedText;
+        var editor = EditorManager.getCurrentFullEditor();

```

Comment:
These two variables seems to be better positioned line 92, using them.

Reply:
Fixed.